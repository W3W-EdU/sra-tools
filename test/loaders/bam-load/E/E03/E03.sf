#bam-load error test #03 : an alunknown ( to bamload ) reference is mentioned
#simulated by the use of the name-override key: 'rname'

r:type=random,name=R1,length=5000

# write all references into one file as FASTA
ref-out:E03.fasta

# write the final SAM-output into this file
sam-out:E03.sam

# write a config-file for bam-load
config:E03.config

p:name=A1,pos=100,rname=R20
p:name=A1,pos=300

#some additional ( correct ) alignments
p:name=B,repeat=50
p:name=B,repeat=50

