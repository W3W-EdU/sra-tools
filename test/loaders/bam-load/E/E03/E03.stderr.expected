err: tools/loaders/bam-loader/bam.c:1956:BAM_FileRefNameIncrementalLookup: row invalid while reading file within alignment module - Parsing SAM RNAME: R20
err: tools/loaders/bam-loader/bam.c:1956:BAM_FileRefNameIncrementalLookup: row invalid while reading file within alignment module - Parsing SAM RNEXT: R1
err: tools/loaders/bam-loader/bam.c:1956:BAM_FileRefNameIncrementalLookup: row invalid while reading file within alignment module - bamread_thread done
err: libs/kdb/wkdb.c:1193:KDBVDrop: row invalid while reading file within alignment module - load failed
